plugins/com.google.cloud.tools.eclipse.appengine.libraries.ui.test/src/com/google/cloud/tools/eclipse/appengine/libraries/ui/MessagesTest.java

```diff
@@ -35,15 +35,8 @@ public void testClientApisTitle() {
 
   @Test
   public void testClientApisDescription() {
-    Assert.assertEquals("Additional jars used by Google Client APIs for Java",
-        Messages.getString("clientapis-description"));
-  }
-
-  @Test
-  public void testDescription() {
-    Assert.assertEquals(
-        "Additional jars commonly used in App Engine standard environment applications",
-        Messages.getString("appengine-description"));
+    Assert.assertEquals("Additional jars for applications using Google Client APIs for Java",
```

Comment:
I don't think there is something called "Google Client APIs for Java". Instead, there is "Google API Client Library for Java", which is different from "Google Cloud Client Library for Java", so I'm inclining to change it to the latter.

Reply:
We have to put them in one container. Do we have to put them in one library though? I.e. can we have multiple libraries attached to the same container?

Reply:
Do you mean: "do we have to edit them all at once in one screen"?  Although the container page is a wizard page, the `ClasspathContainerWizard` only supports a single page.

Reply:
Prior to this PR we have two libraries here that both load into the same Google APIs classpath container:

![image](https://user-images.githubusercontent.com/1005544/32573160-0adb06a2-c49b-11e7-8bb9-6ba1d7852a76.png)

I'm wondering if it's possible to keep that arrangement? Are we required to have a 1:1 relationship between libraries and classpath containers?